.gitignore

```diff
@@ -109,6 +112,9 @@ venv.bak/
 # Rope project settings
 .ropeproject
 
+# VSCode project settings
+.vscode
```

Comment:
What about this/

Reply:
We should be ignoring `.vscode` project settings. Is this bad?